Cargo.toml

```diff
@@ -38,5 +38,42 @@ default-members = [
 [workspace.package]
 readme = "README.md"
 
+# Version Convention: Use major.minor only (e.g., "1.9" not "1.9.1")
+# We accept all compatible patch versions, so specifying patch level is misleading.
+# For exact versions when needed, use "=1.9.1"
+[workspace.dependencies]
+axum = "0.7"
+bitcoin = { version = "0.32", features = ["serde"] }
+clap = { version = "4.5", features = ["derive"] }
+corepc-types = "0.10.1"
+console-subscriber = "0.4"
+dns-lookup = "2.1"
+hex = "0.4"
+kv = "0.24"
+miniscript = { version = "12.3", default-features = false }
```

Comment:
When I try to bump `miniscript` to `v13.0` it panics on a cycle. Any idea why?

Reply:
Not sure. Tried some changes but no effect. Is there any concrete reason to update the `miniscript` version?